test/reuse/helper/errorHandler/logException.spec.js

```diff
@@ -31,19 +31,19 @@ describe("errorHandler - logException - error message with stacktrace by default
 
 describe("errorHandler - logException - generic error message", async function () {
   it("Execution & Verification", async function () {
-    const errorHandler = new ErrorHandler(false);
+    const errorHandler = new ErrorHandler();
     await expect(async () => {
-      errorHandler.logException();
+      errorHandler.logException(undefined, undefined, false);
     }).rejects.toThrow("Failed due to exception in block");
```

Comment:
can we change the error message constant to "Failed due to an exception in the code block"?

Reply:
done